setup.py

```diff
@@ -34,8 +34,8 @@
 import setuptools.command.build as orig_build
 import setuptools.command.develop as orig_develop
 from Cython.Build import cythonize
+from Cython.Distutils import build_ext
```

Comment:
```suggestion
from setuptools.command import build_ext
```

Verified that this solves the issue with `build` not reusing `build_ext` results.

Reply:
Removed entirely so that it uses this as default (and verified that the rebuild issue is taken care of).